Rectangle/Snapping/SnappingManager.swift

```diff
@@ -270,9 +271,21 @@ class SnappingManager {
                         if box == nil {
                             box = FootprintWindow()
                         }
-                        box?.setFrame(.zero, display: false)
-                        box?.makeKeyAndOrderFront(nil)
-                        box?.setFrame(newBoxRect, display: true)
+                        if Defaults.footprintAnimationDurationMultiplier.value > 0 {
+                            if !box!.isVisible, let origin = getFootprintAnimationOrigin(snapArea, newBoxRect) {
```

Comment:
The animation's getting caught up here if the `FootprintWindow` hasn't closed yet (which only happens 0.3 seconds after the `box.close()` call), as it's still `isVisible`, even though it's completely transparent. So if you move a window away from a snapping area and straight back onto it within 0.3 seconds, the animation starts from where the window was when the window was moved away.

Reply:
Good catch. I have created a new PR.